mlir/test/Target/LLVMIR/genx.mlir

```diff
@@ -35,3 +35,9 @@ llvm.func @genx.barrier() {
   genx.barrier
   llvm.return
 }
+
+llvm.func @genx.atomic.cmpxchg.i32(%ptr : !llvm.ptr<i32>, %cmp : i32, %val : i32) -> i32 {
+  // CHECK: call i32 @_Z14atomic_cmpxchgPU3AS1Viii(ptr %0, i32 %1, i32 %2)
```

Comment:
should we check the call returns a value?

Reply:
yes, in this case the return value is used so we can check.

Reply:
done

Reply:
maybe forget to push the change for this?

Reply:
It is in https://github.com/pengtu/llvm-project/pull/13/commits/3fe9e5c4aa0bba18c94387dfeec437ea944ee365.
Currently merging...

Reply:
is fine not to check return value.